Makefile

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# See https://packaging.python.org/en/latest/tutorials/installing-packages/
+# See https://packaging.python.org/tutorials/packaging-projects/
+# python3 -m ensurepip
+# python3 -m pip install --upgrade pip setuptools wheel
+# python3 -m pip install --upgrade build
+# python3 -m pip install --upgrade pylint
+
+default:
+	@echo Nothing to make
+
+################################################################
+# Run pylint over the package
+
+pylint:
+	make -C src/cbmc_starter_kit pylint
```

Comment:
Is this setting for `pylint` useful here? This is the setting used for `cbmc-viewer`
https://github.com/model-checking/cbmc-viewer/blob/master/Makefile#L17-L28
```
	pylint	\
		--disable=unspecified-encoding \
		--disable=consider-using-f-string \
		--disable=duplicate-code \
		--disable=fixme \
		--disable=invalid-repr-returned \
		--disable=too-few-public-methods \
		--disable=too-many-arguments \
		--disable=too-many-branches \
		--module-rgx '[\w-]+' \
```

Reply:
I disagree.  I want the list of things disabled by pylint to be minimal.  Let's keep it as it is.

```
make -n -C src/cbmc_starter_kit pylint
```
```
pylint \
		--disable=missing-module-docstring \
		--disable=missing-function-docstring \
		--disable=duplicate-code \
		--module-rgx '[\w-]+' \
	*.py template-for-repository/proofs/*.py
```